api/docs/release.dox

```diff
@@ -150,6 +150,10 @@ changes:
  - Changed the type of the AArch64 #dr_mcontext_t members svep and ffr to #dr_svep_t.
    This breaks binary compatibility with clients that were built against versions of
    DynamoRIO before this change.
+ - Changed dynamorio::drmemtrace::record_filter_t::record_filter_func_t
+   parallel_shard_filter() interface.  Added a new parameter of type
+   dynamorio::drmemtrace::record_filter_t::record_filter_info_t that allows record_filter
```

Comment:
Looks like the record filter headers are not exported: not put into an export dir and thus never seen by doxygen.  @abhinav92003 was that a deliberate decision?  The record filter interfaces have Doxygen comments, implying some kind of export since we generally don't Doxygen-comment internal interfaces and don't run Doyxgen for anything except the public html.

Reply:
Hmm. I don't remember noting the extra steps for Doxygen in other targets. Maybe I missed it.

Reply:
Doesn't the add_exported_library do all that's needed? https://github.com/DynamoRIO/dynamorio/blob/173fc643fdd604ac06d3688a87677d2211a0f74a/clients/drcachesim/CMakeLists.txt#L194

Oh you mean record_filter.h is missing from there.

Reply:
Doxygen looks at `<builddir>clients/include/drmemtrace/` for the headers.

Reply:
I think `install_client_nonDR_header` puts files in `<builddir>/clients/include/drmemtrace/`, so I added there.
It works.
Done.